apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/privacy-step/index.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import LaunchStep, { Props as LaunchStepProps } from '../../launch-step';
+import './styles.scss';
+
+const PrivacyStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep } ) => {
+	const handleContinue = () => {
+		onNextStep && onNextStep();
+	};
+
+	return (
+		<LaunchStep className="privacy-step">
+			<div className="launch-step__header">
+				<div className="launch-step__heading">
+					<h1 className="launch-step__title">{ __( 'Site privacy', 'full-site-editing' ) }</h1>
+					<p className="launch-step__subtitle">
+						{ __( 'Control who is able to see your site once your launch', 'full-site-editing' ) }
+					</p>
+				</div>
+				<div className="launch-step__actions">
+					<Button isPrimary onClick={ handleContinue }>
+						{ __( 'Continue', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **148** times:
`translate( 'Continue' )` *ES Score: 11*